spyder/plugins/application/container.py

```diff
@@ -278,9 +288,13 @@ def _check_updates_ready(self):
             box.exec_()
             check_updates = box.is_checked()
         else:
+
             if update_available:
-                header = _("<b>Spyder {} is available!</b><br><br>").format(
-                    latest_release)
+                self.application_update_status.set_status_pending()
+
+                header = _("<b>Spyder {} is available!</b> ",
+                           "<i>(you have {})</i><br><br>").format(
+                    latest_release,)
```

Comment:
I think here `__version__` is missing? So maybe @jsbautista this should be:
```suggestion
                    latest_release, __version__)
```

Reply:
Yeah, I would have made it as a suggestion but it required importing `__version__` above, which I couldn't reach